src/Components/Footer/Footer.css

```diff
@@ -1,12 +1,12 @@
 footer {
-    text-align: center;
+    color: white;
     display: flex;
+    font-family: "Anek Latin", sans-serif;
```

Comment:
you can remove the font family from here since it's being set now in the index.css.
also I think you can remove the font-size: large since I think that is also being set in index.css?